okapi/services/attrs/attribute-definitions.xml

```diff
@@ -4070,7 +4070,6 @@ store these attributes in GPX <groundspeak:attribute> elements.
 
     <attr acode="A82" categories="de-cache-types">
         <ocgs id="107" inc="true" />
-        <opencaching schema="OCDE" id="7" />
```

Comment:
I think A82 can be completely removed, as it is never used.

Reply:
Oh. I see. Will remove.

Reply:
Removed in next commit.